providers/shared/references/WAFRule/reference.ftl

```diff
@@ -29,97 +29,3 @@
         } ]
     [#return result]
 [/#function]
-
-[#function formatWAFByteMatchTuples filter={} valueSet={} ]
```

Comment:
Are these all WAF things? Or just code refactoring that has rendered them unnecessary?

Reply:
They were left overs from removing WAF v1 support. They were missed as they weren't obvious in the first round of cleanup